### PR TITLE
(MODULES-4344) bump_to_version rake task

### PR DESCRIPTION
**Pull request by eputnam**
We would like the ability to change the modules version to any version. This new rake task goes around bump levels and accepts any version as a parameter.

### CODE REVIEW

**Comment by eputnam** [comment]
should i add a pin for public_suffix also?

**Comment by bastelfreak** [comment]
we should update the travis matrix I guess. Old ruby is oooooooooold.

**Comment by ekohl** [comment]
@eputnam the pin is merged so please rebase.

File: lib/puppet_blacksmith/modulefile.rb [hunk 1]
```diff
@@ -52,6 +52,13 @@ def version
...
+    def bump_to_version!(new_version)
+      text = File.read(path)
+      text = replace_version(text, new_version)
+      File.open(path,"w") { |file| file.puts text }
+      new_version
+    end
+
     def bump!(level = :patch)
       new_version = increase_version(version, level)
       text = File.read(path)
```
Comment (ekohl) [inline lib/puppet_blacksmith/modulefile.rb]: This can now use ```bump_to_version!(new_version)```

File: lib/puppet_blacksmith/modulefile.rb [hunk 2]
```diff
@@ -52,10 +52,16 @@ def version
...
       text = replace_version(text, new_version)
+      File.open(path,"w") { |file| file.puts text }
+      new_version
+    end
+
+    def bump!(level = :patch)
+      new_version = increase_version(version, level)
+      bump_to_version!(new_version)
       File.open(path, "w") {|file| file.puts text}
```
Comment (ekohl) [inline lib/puppet_blacksmith/modulefile.rb]: The ```bump_to_version!``` already writes the file and returns the new_version so you can remove this and the next line

**Comment by ekohl** [comment]
Needs a rebase now.

**Comment by ekohl** [comment]
Should I squash the commits when merging or do you want to?

**Comment by eputnam** [comment]
i'm happy to!

**Comment by ekohl** [comment]
Thanks!

**Pull request merged**